src/olympia/reviewers/views.py

```diff
@@ -890,8 +898,8 @@ def review(request, addon, channel=None):
         content_review=content_review, count=count,
         deleted_addon_ids=deleted_addon_ids, flags=flags,
         form=form, is_admin=is_admin, is_recommendable=is_recommendable,
-        now=datetime.now(), num_pages=num_pages,
-        pager=pager, reports=reports, show_diff=show_diff,
+        name_translations=name_translations, now=datetime.now(),
+        num_pages=num_pages, pager=pager, reports=reports, show_diff=show_diff,
```

Comment:
ugh, these should just be all on their own lines when there are these many params.  (no requirement to change in this patch, just saying)

Reply:
Yeah, my first version of the patch did that, but then I rebased it a hundred times since and it kept conflicting so I didn't keep it... Next time...